rules/macos/lateral_movement_remote_ssh_login_enabled.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = "Detects the use of systemsetup command to enable remote SSH Login. Verify the context of this action."
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Remote SSH Login Enabled"
+references = [
+    "https://documents.trendmicro.com/assets/pdf/XCSSET_Technical_Brief.pdf",
+    "https://ss64.com/osx/systemsetup.html",
+]
+risk_score = 47
+rule_id = "5ae4e6f8-d1bf-40fa-96ba-e29645e1e4dc"
+severity = "medium"
+tags = ["Elastic", "MacOS"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats